src/common/FrameManager.ts

```diff
@@ -792,6 +792,13 @@ export class Frame {
     return this._mainWorld.executionContext();
   }
 
+  /**
+   * @returns the CDP session associated with the frame.
+   */
```

Comment:
let's add `@internal` and remove the API doc as I am not sure we will need it as a public API